website/sidebars.js

```diff
@@ -187,7 +187,7 @@ const sidebars = {
                   ]
```

Comment:
why are there changes in this page?

Reply:
Which page?

Reply:
sidebar

Reply:
looks like formatting changes. It's the same as the main version. I cross-verified.